test/kubetest/k8s.go

```diff
@@ -718,6 +718,21 @@ func (k8s *K8s) DescribePod(pod *v1.Pod) {
 	}
 }
 
+//ShouldNotRestart checks that pod should not restart
+func (k8s *K8s) ShouldNotRestart(pod *v1.Pod, secondsForWait time.Duration) error {
+	containerStatuses := pod.Status.ContainerStatuses
+	for start := time.Now(); time.Since(start) < time.Second*secondsForWait; {
+		for _, container := range containerStatuses {
+			if container.State.Terminated != nil {
+				return errors.Errorf("The pod %v was restarted", pod.Name)
```

Comment:
Error strings should not start with a capital letter. https://github.com/golang/go/wiki/Errors

Reply:
Should we also put a reason from`state.Terminated` into error?